src/snek5000/output/base.py

```diff
@@ -578,6 +607,35 @@ def write_makefile_usr(self, template, fp=None, **template_vars):
                 with open(makefile_usr, "w") as fp:
                     fp.write(output)
 
+    def write_snakemake_config(self, custom_env_vars=None):
+        """Write the config file in the simulation directory
+
+        Parameters
+        ----------
+
+        custom_env_vars: dict (None)
+            Environment variables used to update the configuration found by
+            :meth:`find_configfile`.
+
+        """
+        if mpi.rank != 0:
+            return
+
+        path_configfile = self.find_configfile()
```

Comment:
We can pass a `host` parameter to `find_configfile` here.

Reply:
Done in d434825f170b939e91f337d2b88c6fc4219676e3